src/views/EmbedView.vue

```diff
@@ -25,6 +26,31 @@ onBeforeMount(() => {
 onMounted(() => {
     log.info(`Map view mounted`)
 })
+
+// as described by this example on our documentation : https://codepen.io/geoadmin/pen/yOBzqM?editors=0010
+// our embed view should send a message (see https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage)
+// when a feature is selected while in embedded mode, so that the parent can get the selected feature(s) ID(s)
+watch(selectedLayerFeatures, () => {
+    if (selectedLayerFeatures.value.length > 0) {
```

Comment:
Just a minor thing, a general idea: we used to use the concept of early returns. This way, the function becomes a bit more readable, in my opinion. Less indentation.

so instead of having
```
function foo(param) {
   if (isGood(param)) {
       // do all the things
   }
}
```
we proposed to do it like this
```
function foo(param) {
   if (!isGood(param)) {
     return
   }
   
   // do all the things here
}
```
Just an idea I want to drop here :)

Reply:
I also quite like this approach, some other don't, so I complied and didn't use it much :wink: 
but you have my vote!

Reply:
OK Cool :star_struck: Let's see if it changes some time :wink:

Reply:
Early returns at the top of the function are ok and good in such example as above or like here, but I would not do returns in the middle of a function like this

```javascript
function foo(param) {
  if (!isGood(param)) {
    return
  }
  // do something
  if (...) {
    return
  }
  // do something else
   
  // final return
  return
}
```
This would be a very poor and hard code to read as some returns would be hidden in the middle of the function.

Reply:
@ltshb Agree!